app/jobs/import_versions_job.rb

```diff
@@ -100,7 +100,9 @@ def import_record(record, row)
     elsif !Archiver.already_archived?(version.uri) || !version.version_hash
       result = Archiver.archive(version.uri, expected_hash: version.version_hash)
       version.version_hash = result[:hash]
+      version.length = result[:length]
```

Comment:
Yep, the more I think about it, the more sure I am this should be named `content_length` instead of `length`. The current name is too confusing since it often has a specialized meaning in many languages or tools.